core/commands/storage.go

```diff
@@ -1277,31 +1276,35 @@ This command updates host information and broadcasts to the BTFS network.`,
 			return fmt.Errorf("storage host api not enabled")
 		}
 
-		sp, spFound := req.Options[hostStoragePriceOptionName].(uint64)
-		bp, bpFound := req.Options[hostBandwidthPriceOptionName].(uint64)
-		cp, cpFound := req.Options[hostCollateralPriceOptionName].(uint64)
+		sp, spFound := req.Options[hostStoragePriceOptionName].(float64)
+		bp, bpFound := req.Options[hostBandwidthPriceOptionName].(float64)
+		cp, cpFound := req.Options[hostCollateralPriceOptionName].(float64)
 		bl, blFound := req.Options[hostBandwidthLimitOptionName].(float64)
-		stm, stmFound := req.Options[hostStorageTimeMinOptionName].(uint64)
+		stm, stmFound := req.Options[hostStorageTimeMinOptionName].(float64)
 
 		n, err := cmdenv.GetNode(env)
 		if err != nil {
 			return err
 		}
 
 		rds := n.Repo.Datastore()
+		peerId := n.Identity.Pretty()
 
-		selfKey := storage.GetHostStorageKey(n.Identity.Pretty())
+		ns := new(hubpb.SettingsData)
+		selfKey := storage.GetHostStorageKey(peerId)
 		b, err := rds.Get(selfKey)
-		// If key not found, create new
-		if err != nil && err != ds.ErrNotFound {
+		if err == ds.ErrNotFound {
+			// If key not found, get from remote
+			if n, err := GetSettings(req.Context, cfg.Services.HubDomain, peerId, rds); err != nil {
+				log.Error(err)
```

Comment:
`return err`

Since this is a manual command, you can let it fail and report err